electrum_dash/coins.py

```diff
@@ -64,3 +73,10 @@ class FiroTestnet(Firo):
     PRE_MTP_BLOCKS = 1
     AFTER_PROGPOW_BLOCK_HEIGHT = 37305
     PROGPOW_START_TIME = 1630069200
+
+class FiroRegtest(Firo):
+    PRE_MTP_BLOCKS = 2147483647             # INT_MAX 0xefffffff
```

Comment:
also why `0xefffffff`? `INT_MAX` is `0x7fffffff`

Reply:
Thanks for pointing this out
![image](https://github.com/firoorg/electrum-firo/assets/126670673/89ca4a7c-82a3-4968-85af-1ba0a093c950)

The comment is wrong the number is correct. msb not set. Will change the comment also. My tests so far showing no need for the short-circuit logic above but this time am testing `0x7fffffff` cases.

Reply:
There is one minor edge case in:
https://github.com/firoorg/electrum-firo/blob/97873167768250e60ade56a17f717a0125e2ba97/electrum_dash/coins.py#L35

if `height` is 2147483647 (INT_MAX) then 120 is the returned size of the header. 
It is regtest and never going to hit that height

So I will clean up my code and make a new commit with the short-circuit code removed and updated comments

Reply:
Fixed in: https://github.com/firoorg/electrum-firo/pull/41/commits/0fdea46083aff75ab30ab5499c59a537c11db6de